src/Units/UnitModel.java

```diff
@@ -53,6 +53,13 @@ public abstract class UnitModel implements Serializable, ITargetable, UnitModLis
   public static final long AIR_TO_AIR      = 1 << 22;
   public static final long TRANSPORT       = 1 << 23;
 
+  // Calculated properties so I can be lazy
+  public static final long DIRECT          = 1l << 50; // Has only weapons for 1 range
+  public static final long INDIRECT        = 1l << 51; // Has weapons for 2+ range
+  // Sins against game design
+  public static final long GREYFIELD_LIKES = 1l << 62; // For Seaplanes
```

Comment:
Sigh. Well, at least you are going in with both eyes open.

Reply:
The only other real option is filtering on the name, and that seems bad too.
...I guess you could look at fuel/ammo numbers or something, but bleh.

Reply:
I've realized also that Seaplanes on cart can load the COU from an airport (but that's not true here).
Should I just reuse this flag to fix that?
Or maybe I can make the check "can build any unit type, and can repair this unit"?

Reply:
> filtering by name

As a one-off for a CO with a special affinity for a particular unit, I'd probs allow it.

I'm also fine with adapting the condition to allow for properties that repair to avoid more heinous hijinks.